EIPS/eip-6963.md

```diff
@@ -75,6 +75,16 @@ data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12
 
 The `icon` string MUST be a data URI as defined in [RFC-2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur.
 
+#### RDNS
+
+The **`rdns`** (Reverse-DNS) property serves to provide an identifier which DApps can rely on to be stable between sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions.
+The Reverse-DNS convention implies that the value should start with a reversed DNS domain name controlled by the Provider. The domain name should be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
+
+- The DNS part of the `rdns` value MUST BE a valid [RFC-1034] Domain Name;
+- The DNS part of the `rdns` value SHOULD BE an active domain controlled by the Provider;
+- DApps MAY reject the Providers which do not follow the Reverse-DNS convention correctly;
+- DApps MUST NOT use the `rnds` value for feature detection;
```

Comment:
Should the spec also prescribe against prioritization?
For example, a DApp with a Connect Screen that chooses to show some wallet options at the top, and chooses to hide other wallets behind the "show more" button.
Such UI patterns _may_ lead to the "browser wars" problem, which this particular proposal is trying to avoid in the first place.

Reply:
I believe that falls under "economic behaviors" and definitely hard to banish with a MUST NOT.  I would say it's better to phrase any guidance in technical terms and, at most, remind implementers that such design decisions create incentives for bad security, i.e. impersonation.  For example:

```suggestion
- DApps SHOULD NOT use the `rnds` value for feature detection as these are self-attested and prone to impersonation or bad incentives without an additional verification mechanism; feature-discovery and verification are both out of scope of this interface specification.
```

Reply:
Good point!